app/build.gradle.kts

```diff
@@ -352,24 +352,44 @@ configurations.forEach { configuration ->
   configuration.exclude("com.google.protobuf", "protobuf-lite")
 }
 
+fun runCommand(command: String) {
+    val os = System.getProperty("os.name").toLowerCase()
+    if (os.contains("win")) {
+        ProcessBuilder("cmd", "/c", "start", "cmd", "/k", command).start()
+    } else if (os.contains("nix") || os.contains("nux") || os.contains("bsd")) {
+        ProcessBuilder(
+            "sh",
+            "-c",
+            """
+                x-terminal-emulator -e '$command' ||
+                gnome-terminal -- bash -c '$command; exec bash' ||
+                konsole -e '$command'
+                """.trimIndent()
+        ).start()
```

Comment:
As you can see at the bottom of the [Java documentation for ProcessBuilder.start()](https://docs.oracle.com/javase/8/docs/api/index.html?java/lang/ProcessBuilder.html), the method can throw an `IOException`: "Starting an operating system process is highly system-dependent. Among the many things that can go wrong are:

    The operating system program file was not found.
    Access to the program file was denied.
    The working directory does not exist. 

In such cases an exception will be thrown. The exact nature of the exception is system-dependent, but it will always be a subclass of [IOException](https://docs.oracle.com/javase/8/docs/api/java/io/IOException.html).". Consider handling it with a `catch (e: IOException)`, and other errors with `catch (e: Exception)`.

Reply:
Indeed, that's intentional :)
If an error occurs when running the command, it should be shown to the developper running it.